website/source/docs/job-specification/reschedule.html.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: "docs"
+page_title: "reschedule Stanza - Job Specification"
+sidebar_current: "docs-job-specification-reschedule"
+description: |-
+  The "reschedule" stanza specifies the group's rescheduling strategy upon task failures.
+  The reschedule strategy can be configured with number of attempts and a time interval.
+  Nomad will attempt to reschedule failed allocations on to another node only after
+  any applicable [restarts](docs/job-specification/restart.html) have been tried.
+---
+
+# `reschedule` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> **reschedule**</code>
+    </td>
+    <td>
+      <code>job -> group -> **reschedule**</code>
+    </td>
+  </tr>
+</table>
+
+The `reschedule` stanza specifies the group's rescheduling strategy.
+It can be configured with number of attempts and a time interval. If
```

Comment:
Is the "if omitted" part true? It defaults to reschedule, so they have to instead set the attempts = 0

Reply:
good catch, will fix